rts/Map/SMF/ROAM/RoamMeshDrawer.cpp

```diff
@@ -33,6 +33,11 @@ LOG_REGISTER_SECTION_GLOBAL(LOG_SECTION_ROAM)
 #define LOG_SECTION_CURRENT LOG_SECTION_ROAM
 
 
+//tessmode can be 1,2,(3 is LOAM)
+#define RETESSELLATE_MODE 3
+#define RETESSELATE_TO_FREE_INVISIBLE_PATCHES 2
```

Comment:
Inconsistent spelling (see the line above)
```suggestion
#define RETESSELLATE_TO_FREE_INVISIBLE_PATCHES 2
```